src/mem/largealloc.h

```diff
@@ -61,7 +68,7 @@ namespace snmalloc
      * Primitive allocator for structure that are required before
      * the allocator can be running.
      ***/
-    void* alloc_chunk(size_t size)
+    void* alloc_chunk(size_t size, size_t alignment = 64)

```

Comment:
Do we ever not know the alignment statically?  If not, then we might want to make alignment a template parameter.

Reply:
Good point. Will fix.